migtests/scripts/run-test.sh

```diff
@@ -168,6 +168,11 @@ main() {
 	expected_file="${TEST_DIR}/export_data_status-report.json"
 	actual_file="${EXPORT_DIR}/reports/export-data-status-report.json"
 
+	if [ "${EXPORT_TABLE_LIST}" != "" ]
```

Comment:
Where are we changing the invocation of this test where we actually pass a table-list?

Reply:
In the `pg-migtests.yml`

Reply:
Ah okay, this is already the case that we use table-list for this test-case. got it!